docs/interfaces/action-service.txt

```diff
@@ -5,15 +5,21 @@ Exception ActionException(message: string);
 
 /**
  * Executes multiple actions in the conext of the user given by the user_id.
- * All actions are successful if no exception is thrown, else no action was
- * processed.
+ * All actions are processed independently.
+ *
+ * TODO: Check if we need an interface or flag to process actions all together.
```

Comment:
This depends on the actions you implement. But I'm sure, that we will need this. I think about actions, that cannot be covered with a catch-all-update-action, like resetting a motion's state. In a client update view, a user may change some properties of the motion and resets the state. This would result in two _coupled_ actions, that must be performed together atomically.

Reply:
So let's see when such actions appear. Until this we can merge this PR.

Reply:
Ok